core/src/components/popover/test/util.spec.ts

```diff
@@ -17,45 +17,45 @@ describe('isTriggerElement', () => {
 
 describe('getIndexOfItem', () => {
   it('should return the correct index in an array of ion-items', () => {
-    const array = createArrayOfElements(['ion-item', 'ion-item', 'ion-item']);
+    const array = createArrayOfElements(['ion-item', 'ion-item', 'ion-item']) as HTMLIonItemElement[];
```

Comment:
Optional: You could get fancy with this by adding a generic type parameter to `createArrayOfElements`. Not super important, just feels nicer than casting 😄

Reply:
I tried looking into this originally, but later usages passes an item and a div. So we'd have to do potentially a generic that works as a tuple.. and that got way more involved than seemed worth it here.